shell/browser/native_window.h

```diff
@@ -149,6 +149,9 @@ class NativeWindow : public base::SupportsUserData,
   virtual std::string GetAlwaysOnTopLevel() = 0;
   virtual void SetActive(bool is_key) = 0;
   virtual bool IsActive() const = 0;
+  virtual void RemoveChildWindow(NativeWindow* child) = 0;
```

Comment:
This method doesn't need to be in `NativeWindow`; it can removed from here and declared non-virtual in 
 `NativeWindowMac`

Reply:
oh good call - will do!